app/helpers/advanced_search_helper.rb

```diff
@@ -20,7 +20,11 @@ def filter_boolean_value(filter_action, tag_type, tag_id)
       @search.send("#{tag_type}_ids").present? &&
         @search.send("#{tag_type}_ids").include?(tag_id)
     else
-      @search.excluded_tag_ids.present? && @search.excluded_tag_ids.include?(tag_id)
+      value = @search.excluded_tag_ids.present? && @search.excluded_tag_ids.include?(tag_id)
+      if @search.respond_to?(:excluded_bookmark_tag_ids)
```

Comment:
This helper is also used on WorkSearchForm which doesn't have the `excluded_bookmark_tag_ids` accessor.